app/src/main/res/menu/manage_pledge_plot_selected.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
@jlplks there is a menu already, I much likely prefer to identify where that menu is, and hide the options not available rather than creating a new one

Reply:
the way the menu options are managed inside the class is rendering differents xmls depending on the conditions, that's the reason for managePledgeMenu. That's why I opted for that way, do you prefer that I did it in another way?

Reply:
Added incremental to backing and added a test